assets/stories/skate-components/map/tooltip.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { MapTooltip } from "../../../src/components/map/tooltip"
+import type { Meta, StoryObj } from "@storybook/react"
+import { inMapDecorator } from "../../../.storybook/inMapDecorator"
+import { Polyline } from "react-leaflet"
+
+const meta = {
+  component: MapTooltip,
+  args: {
+    title: "Click to start detour",
+    permanent: true,
+  },
+  argTypes: {
+    permanent: { table: { disable: true } },
+  },
```

Comment:
question(non-blocking): why disable this arg?

Reply:
Without the `key` prop, this toggle doesn't actually do anything (because leaflet `Tooltip`'s don't re-render properly).

(With the `key` prop, this toggle makes the tooltip appear and disappear with a pretty glitchy animation 😂)

The reason I added `permanent` at all was so that in Storybook, when you navigate to the page for this component, there's actually a tooltip visible. Otherwise, you'd have to hover in order to see it, which is kind of annoying.